src/main.ts

```diff
@@ -4,6 +4,7 @@ dotenv.config({ path: `.env.${process.env.NODE_ENV}` });
 
 import { ValidationPipe, Logger } from '@nestjs/common';
 import { NestFactory, Reflector } from '@nestjs/core';
+import * as process from 'process';
```

Comment:
Nest does manage the environment, use ConfigService (or configuration as fallback) to get the environment values.

Reply:
I'm using the `configuration` to get the environment and validate when using the middleware logger

Reply:
It appears as new line on this PR, how is that?

Reply:
because I did the `configuration` import below it, but anyway I removed it, I'm already only using the `configuration`